squidb-processor/src/com/yahoo/squidb/processor/writers/ModelFileWriter.java

```diff
@@ -115,8 +116,6 @@ private void emitImports() throws IOException {
     }
 
     private void beginClassDeclaration() throws IOException {
-        writer.writeComment("Generated code -- do not modify!");
-        writer.writeComment("This class was generated from the model spec at " + modelSpec.getModelSpecName());
```

Comment:
It might be nice to still have these lines somewhere as it's a good warning to someone who opens the file